modules/installation/pages/pubcloud-setup.adoc

```diff
@@ -0,0 +1,82 @@
+[[public-cloud-setup]]
+= Public Cloud Setup
+
+Public Cloud providers pre-install {productname}, so you do not need to perform any installation steps.
+However, {productname} Server needs to be registered with {scc} to receive updates before you can log in.
+
+For detailed instructions on registering {productname} to {scc}, see xref:installation:server-setup.adoc[].
+
+When you have registered, all {sle} modules will be activated.
+You will also need to activate the public cloud module.
+
+
+
+.Procedure: Activating the Public Cloud Module
+
+. On the {productname} Server, from the desktop, open the {yast} management tool, and navigate to menu:Software[Software Repositories].
+. Click btn:``Add`` and select [guimenu]``Extensions and Modules from Registration Server``.
```

Comment:
```suggestion
. Click btn:[Add] and select [guimenu]``Extensions and Modules from Registration Server``.
```
Elsewhere we use this markup with ``btn``.  I do not know whether that's identical...

Reply:
My brain still can't work out btn syntax. You'd think I'd have it by now, huh? :wink: